src/App.js

```diff
@@ -1,12 +1,12 @@
-// eslint-disable-next-line import/order
 import React from 'react';
-// eslint-disable-next-line import/order
-import CardMenu from './components/CardMenus/CardMenus';
+
+import ShopButton from './components/ShopButton/ShopButton';
+
 const App = () => {
   return (
-    <div>
-      <CardMenu />
-    </div>
+    <>
+      <ShopButton />
```

Comment:
What does that shop button means .. and it should not be in app just use it there for testing purpose and wait for the header works to merge and use it in header .. Please do some brain storming when naming things as it need time to name things rightly ..